string pose_name
string pose_frame_id